polar-core/src/vm.rs

```diff
@@ -895,10 +945,7 @@ impl PolarVirtualMachine {
     /// Remove all bindings after the last choice point, and try the
     /// next available alternative. If no choice is possible, halt.
     fn backtrack(&mut self) -> Result<()> {
-        if self.log {
-            self.print("⇒ backtrack");
-        }
-        self.log("BACKTRACK", &[]);
+        self.log_with(LogLevel::Trace, || "BACKTRACK", &[]);
```

Comment:
Looks like we probably logged `BACKTRACK` _and_ `⇒ backtrack` on `POLAR_LOG=trace` for some period of time 🤦. That probably happened when we cleaned up the RUST_LOG/trace stuff a little while ago.

Anyway, just verifying that before this PR `POLAR_LOG=on` would spit out a `BACKTRACK` and after this PR it'll be silent unless `POLAR_LOG=trace`. Is that the desired behavior?

Reply:
expected behavior